module/VuFind/src/VuFind/Auth/Manager.php

```diff
@@ -271,6 +282,37 @@ public function supportsConnectingLibraryCard($authMethod = null)
             && $this->getAuth($authMethod)->supportsConnectingLibraryCard();
     }
 
+    /**
+     * Is persistent login supported?
+     *
+     * @param User $user optional; check user auth method in database
```

Comment:
This parameter is never used. Is it really needed?

Reply:
Not needed, removed.